### PR TITLE
Properly initialize viper library when checking for master components

### PR DESCRIPTION
**Pull request by simar7**
This PR includes:
* Properly initializes the viper library config when checking for master components.
* Adds some additional test coverage to help guard against future regressions.

Fixes: https://github.com/aquasecurity/kube-bench/issues/339

Signed-off-by: Simarpreet Singh <simar.singh@aquasec.com>

### CODE REVIEW

File: cmd/common.go [hunk 1]
```diff
@@ -242,7 +242,10 @@ func loadConfig(nodetype check.NodeType) string {
 func isMaster() bool {
 	glog.V(2).Info("Checking if the current node is running master components")
 	masterConf := viper.Sub(string(check.MASTER))
-	components, err := getBinaries(masterConf)
+	if masterConf == nil {
```
Comment (robertojrojas) [inline cmd/common.go]: We should add a log statement indicating why isMaster is returning `false` in this situation. It'll help with troubleshooting later on. I think glog.V(2).Info(...) should be fine.

File: cmd/common_test.go [hunk 2]
```diff
@@ -110,3 +113,135 @@ func TestNewRunFilter(t *testing.T) {
 	})
 
 }
+
```
Comment (robertojrojas) [inline cmd/common_test.go]: I like the fact the testing using file data. I think it'll be better to have the contents to be in its own files, like `master_only.yaml`, `node_only.yaml`, `master_node.yaml`. The benefit I see of having external files is that that it'll be easier to check of file format and modifications as these files change later one.

File: cmd/common.go (hunk 1 above)
Comment (simar7) [reply cmd/common.go]: Added.

File: cmd/common_test.go (hunk 2 above)
Comment (simar7) [reply cmd/common_test.go]: Addressed.

**Pull request merged**